community.yml

```diff
@@ -41,6 +41,7 @@
     - { role: gotify, tags: ['gotify'] }
     - { role: guacamole, tags: ['guacamole'] }
     - { role: handbrake, tags: ['handbrake'] }
+    - { role: healthchecks, tags: ['healthchecks'] }
```

Comment:
Why has this been removed? This looks to still be a part of develop so not sure it should have been removed?

Reply:
I misunderstood here. I can see you added this in, inline with the rest of develop but you haven't added in the healthchecks role folder, so when it tries to run this it errors. Please add the healthchecks role folder in.